koku/api/report/aws/view.py

```diff
@@ -27,10 +27,10 @@ class AWSView(ReportView):
     """AWS Base View."""
 
     permission_classes = [AwsAccessPermission]
-    provider = 'aws'
-    _serializer = QueryParamSerializer
-    _query_handler = AWSReportQueryHandler
-    _tag_handler = [AWSTagsSummary]
+    provider = 'AWS'
+    serializer = QueryParamSerializer
+    query_handler = AWSReportQueryHandler
+    tag_handler = [AWSTagsSummary]
```

Comment:
This bit is me fixing an incorrect choice I made in a previous PR. These attributes are not private, particularly with how `QueryParameters` needs to access them. So, I'm removing the leading `_` and making these public attributes.